### ISSUE TITLE
ol lifecycle error

### ISSUE THREAD

**Issue by scadergit** [ghettovoice/vuelayers#461]
Getting the following error after upgrading to VL 0.12.0-rc26 (which bumped OL to 6.9.0). I attempted to review the [previous lifecycle issue](https://github.com/ghettovoice/vuelayers/issues/390), but that [pull request](https://github.com/ghettovoice/vuelayers/pull/369/files) had quite a few changes. Would this be similar? If related, we use `v-else` making `v-show` not possible. I haven't been able to identify the exact component yet. I'll keep looking into it, but haven't dealt with this aspect of OpenLayers yet.

https://github.com/ghettovoice/vuelayers/blob/d0aca44d0af12c4561962b58dfd293f191c12dbd/src/mixins/ol-cmp.js#L765

![image](https://user-images.githubusercontent.com/17500494/142296987-52cfaac5-05da-4681-8ce7-fd1b16a5c94b.png)

**Comment by scadergit** [comment]
So the component id makes it obvious this is affecting the VL view component, but oddly the uuid doens't match up to the vue inspector. You'd think they would be the same? Not sure why it would ever create more than one OL View component.

![image](https://user-images.githubusercontent.com/17500494/142336953-a30f5268-121d-4081-b20c-6b823b3e5721.png)

![image](https://user-images.githubusercontent.com/17500494/142337040-6c648b60-f853-472b-ac02-6f8692f1eed8.png)

**Comment by scadergit** [comment]
Turns out the id in every error is the id of the VLMap component, which I find interesting as `vmName` implies it would have been the VLView component.

**Comment by ghettovoice** [comment]
Hello @scadergit ,
can you dump here a some simplified code sample of your app? 

I'm trying to reproduce this issue and need to understand what lead to it. 
As I see from your first comment, you are using `v-else` somewhere. Is it conditional render of `vl-map` component?

**Comment by wendellwt** [comment]
I had the same issue with the "ol object lifecycle canceled" and was trying to help you, but the problem went away.  All I did was make an entire copy of the directory, and I did not get that error.  The specific steps were:

```
$ cd orig_tree
$ tar cvf - * | ( cd ../no_lifecycle_error ; tar xvf - )
$ cd ../no_lifecycle_error/vuelayers_v12/
$ npm run serve
```
and in another window:
```
$ cd no_lifecycle_error/vuelayers_v12/src
$ code .
```
Running what seemed to be the exact same code from the `no_lifecycle_error/vuelayers_v12` directory ran fine.  Running from the `orig_tree/vuelayers_v12` gave the error.  Switching back and forth kept giving the same results (the error always happened when running from `orig_tree`, and always did _not_ happen when running from `no_lifecycle_error` tree).

Sorry, but I was not able to reproduce the error, but I was able to at least make it go away.
